src/event/event.service.ts

```diff
@@ -211,11 +212,12 @@ export class EventService {
 
     if (search) {
       eventQuery.andWhere(
-        `(event.name LIKE :search OR 
-          event.description LIKE :search OR 
-          event.location LIKE :search OR 
-          CAST(event.lat AS TEXT) LIKE :search OR 
-          CAST(event.lon AS TEXT) LIKE :search)`,
+        `(event.name LIKE :search OR
+          event.slug LIKE :search OR 
+          event.description LIKE :search OR
+          CAST(event.type AS TEXT) LIKE :search OR
+          CAST(event.status AS TEXT) LIKE :search OR
```

Comment:
search by type and status as well as slug isn't needed. Only name and filtering by status with visibility